python/setup.py

```diff
@@ -4,11 +4,12 @@
 exec(open('./metaparticle/version.py').read())
 
 setuptools.setup(
-    name='metaparticle',
+    name='metaparticle_pkg',
```

Comment:
just out of curiosity, why go with `metaparticle_pkg` when `metaparticle` is available?

Reply:
because there will eventually be a `metaparticle_sync` library too (https://github.com/metaparticle-io/sync)